src/routes/device-type/add.ts

```diff
@@ -12,7 +12,22 @@ interface Request {
       name: string;
       pinFieldConfigurations: FieldConfigurationsToServer;
       defaultPinName?: string;
+      charts?: {
+        title: string | null;
+        series: {
+          quantityHashId: string;
+          color: string;
+        }[];
+      }[];
     }[];
+    charts: {
+      title: string | null;
+      series: {
+        channelIndex: number;
+        quantityHashId: string;
+        color: string;
+      }[];
+    }[] | null;
```

Comment:
I don't think the `null` is needed: client can just set `[]`. I would like to make this object optional though, for backwards compatability.

Reply:
Both charts objects in deviceType.add should be optional for backwards compatibility.